example_configurations/bug_conf.ini

```diff
@@ -0,0 +1,53 @@
+[General]
```

Comment:
Why is this file needed? Should we replace `faas_test_hyperopt_sfs.ini` with this? What can we test with this file that we cannot with the other?


Long story short, I'd keep only one of them since they appear to be redundant. Let's choose which one

Reply:
I used it to have a feedback when the xgboost was training, not sure why though: now removed